### ISSUE TITLE
It is not possible to create a Native App Selection of MathML content

### ISSUE THREAD

**Issue by SuzanneTaylor** [nvaccess/nvda#20643]
### Brief summary

Native app selection mode (NVDA Shift F10) does not work inside of MathML. 

### You Can:
- You can copy the current math node using Control C.
- You can even decide the format of what is placed on the clipboard using settings.

### But you can't:
- Create a selection within the MathML that a Web application can detect using JavaScript and work with. 

### Why this is important:
Websites might provide tutorial information about the particular selection, allow users to annotation the particular bit of math, etc

### Steps to reproduce

1. With NVDA running, open Chrome, Edge or Firefox
2. Go to [Math Selection Text Page](https://thingsentertainment.net/Shared/BugReportFiles/HowToSayPolynomialPieces.html)
3. _(Consider: Shift with arrow keys can't be used as those commands have other purposes: https://download.nvaccess.org/documentation/userGuide.html#ReadingMath)_
4. Turn on Native App Selection Mode: NVDA Shift F10. Try NVDA F9, navigate, NVDA F10 to try to create a selection. Or just press NVDA F9, NVDA F10 to try to select the current node.

### Actual behavior

NVDA just says "No text to copy"

### Expected behavior

- NVDA F9 followed immediately by NVDA F10 could select the current node.
- NVDA F9, navigate, NVDA F10 could select all nodes between the two nodes marked by pressing the mentioned keys.

### NVDA type

installed copy

### NVDA version

2026.1 (2026.1.0.55743)

### Windows version

Version	10.0.26200 Build 26200

### Name and version of other software in use when reproducing the issue

This is the same in Chrome, Edge, Firefox, but just tested in:
Chrome Version 151.0.7922.109 (Official Build) (64-bit)

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by SaschaCowley** [comment]
For technical reasons it is unlikely that we will be able to support selection of individual parts of MathML in all cases. This may be possible in some cases using an approach similar to 20372.

**Comment by SuzanneTaylor** [comment]
thank you @SaschaCowley !
